vaadin-flow-components-shared-parent/vaadin-flow-components-test-util/src/main/java/com/vaadin/tests/validation/AbstractValidationIT.java

```diff
@@ -46,6 +46,7 @@ protected void assertValidationCount(int expected) {
                 .parseInt($("div").id(SERVER_VALIDATION_COUNTER).getText());
         Assert.assertEquals("The field should have validated " + expected
                 + " times on the server-side", expected, actual);
+        resetValidationCount();
```

Comment:
**note:** Implicitly resetting the validation count after each assertion simplifies tests. I haven't come up with a use-case where it would be useful to preserve the count after the assertion.